test/functional/test_framework/messages.py

```diff
@@ -405,6 +405,14 @@ def serialize(self):
         r += self.nInflationKeys.serialize()
         return r
 
+    # serialization used in taproot sighash
+    def tap_sighash_serialize(self):
```

Comment:
Can you rename this (and change the comment) to indicate that it is specifically for asset issuance data?

Reply:
It is inside the class `CAssetIssuance`, so it is implicit that it is for asset issuance?

Reply:
edited the name to `taphash_asset_issuance_serialize`